### PR TITLE
K8SPXC-1363 Update images and versions for 1.15.0 release

### PR DESCRIPTION
**Pull request by ptankov**
[K8SPXC-1363](https://jira.percona.com/browse/K8SPXC-1363) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
*Short explanation of the problem.*

**Cause:**
*Short explanation of the root cause of the issue if applicable.*

**Solution:**
*Short explanation of the solution we are providing with this PR.*

**CHECKLIST**
---
**Jira**
- [ ] Is the Jira ticket created and referenced properly?
- [ ] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?
- [ ] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported PXC version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

[K8SPXC-1363]: https://perconadev.atlassian.net/browse/K8SPXC-1363?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/cr-minimal.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
   haproxy:
     enabled: true
     size: 1
-    image: perconalab/percona-xtradb-cluster-operator:main-haproxy
+    image: percona/percona-xtradb-cluster-operator:1.15.0-haproxy
```
Comment (hors) [inline deploy/cr-minimal.yaml:28]: We can use the official percona HaProxy image now

File: deploy/cr.yaml [hunk 2]
```diff
@@ -201,7 +201,7 @@ spec:
   haproxy:
     enabled: true
     size: 3
-    image: perconalab/percona-xtradb-cluster-operator:main-haproxy
+    image: percona/percona-xtradb-cluster-operator:1.15.0-haproxy
```
Comment (hors) [inline deploy/cr.yaml:204]: We can use the official percona HaProxy image now

File: deploy/cr.yaml [hunk 3]
```diff
@@ -377,7 +377,7 @@ spec:
   proxysql:
     enabled: false
     size: 3
-    image: perconalab/percona-xtradb-cluster-operator:main-proxysql
+    image: percona/percona-xtradb-cluster-operator:1.15.0-proxysql
```
Comment (hors) [inline deploy/cr.yaml:380]: We can use the official percona ProxySQL image now

**Review by hors — CHANGES_REQUESTED** [review]

**Review by hors — APPROVED** [review]

**Review by eleo007 — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/affinity-8-0.log) |
| auto-tuning-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/auto-tuning-8-0.log) |
| cross-site-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/demand-backup-8-0.log) |
| haproxy-5-7 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/haproxy-5-7.log) |
| haproxy-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/haproxy-8-0.log) |
| init-deploy-5-7 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/init-deploy-5-7.log) |
| init-deploy-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/init-deploy-8-0.log) |
| limits-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/limits-8-0.log) |
| monitoring-2-0-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/one-pod-5-7.log) |
| one-pod-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/one-pod-8-0.log) |
| pitr-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/pitr-8-0.log) |
| pitr-gap-errors-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/pitr-gap-errors-8-0.log) |
| proxy-protocol-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/proxysql-sidecar-res-limits-8-0.log) |
| pvc-resize-5-7 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/pvc-resize-5-7.log) |
| pvc-resize-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/pvc-resize-8-0.log) |
| recreate-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/scaling-proxysql-8-0.log) |
| scaling-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/scaling-8-0.log) |
| scheduled-backup-5-7 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/scheduled-backup-8-0.log) |
| security-context-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/security-context-8-0.log) |
| smart-update1-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/smart-update1-8-0.log) |
| smart-update2-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/smart-update2-8-0.log) |
| storage-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/upgrade-consistency-8-0.log) |
| upgrade-haproxy-5-7 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/upgrade-haproxy-5-7.log) |
| upgrade-haproxy-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-5-7 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/upgrade-proxysql-5-7.log) |
| upgrade-proxysql-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/upgrade-proxysql-8-0.log) |
| users-5-7 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/users-5-7.log) |
| users-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/users-8-0.log) |
| validation-hook-8-0 | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1780/b254b0b1/validation-hook-8-0.log) |
| We run 0 out of 41|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1780/commits/b254b0b1233ff766850b982eb849ccb64ffdb5cb
image: `perconalab/percona-xtradb-cluster-operator:PR-1780-b254b0b1`

**Pull request merged**